config/groovy/common.groovy

```diff
@@ -215,8 +250,33 @@ class common {
                 println color("uncommitted changes. Skipping.", Ansi.YELLOW)
             } else {
                 println color("updating $itemType $itemName", Ansi.GREEN)
+                File targetDirFetchHead = new File("$targetDir/.git/FETCH_HEAD")
+                def timeLimit = use(groovy.time.TimeCategory){ 10.minute }
+                if (skipRecentUpdates && isRecentlyUpdated(targetDirFetchHead, timeLimit)){
```

Comment:
I just realized in a newly cloned module the FETCH_HEAD may not exist. I need to add some sort of check to make sure the file exists before running `isRecentlyUpdated()`

Reply:
reading stuff directly out of `.git/` makes me anxious.

I won't say it's _never_ a good idea. In practice, those sorts of files usually have pretty stable formats. But if you can do it through your git client — which in this case looks like is Grgit wrapping JGit — do prefer that. Then they can be the ones responsible for handling whether or not files exist and what settings are in them.

Reply:
I'm not actually reading the file, just checking the last time the file was modified by git.

Reply:
I'd still vote for removing this "feature" 🤷

Reply:
So I'm not married to the concept of that feature personally, I thought it could be a cool bit of utility if it would work out easily enough, but if it is becoming too complex I'm not at all opposed to going without :-)

I just retried it, ran `groovyw module update-all` and one module updated. Ran `groovyw module update-all --continue` and all skipped. So it seemed like it worked 👍 Whether it actually is meaningful at this point I'm less sure about. 

We had more cases in the past where an `update-all` in a mega-workspace could hit _one_ module where the remote was gone (say the user had been testing some other users module and it had been deleted on GitHub) which would break the whole update. But I just tried to break a module's Git config and the process caught that gracefully, skipped it, and continued on its own.

Still, that could just be one case, I'm not sure if there might be others. It seems like it works perfectly at this point and it is optional if passing in `--continue` - maybe it'll still be useful?